rdbms/src/main/java/org/spine3/server/storage/jdbc/command/query/CommandStorageQueryFactory.java

```diff
@@ -46,6 +83,12 @@ public InsertCommandQuery newInsertCommandQuery(CommandId id, CommandStorageReco
         return builder.build();
     }
 
+    /**
+     * Returns a query that updates {@link CommandStorageRecord} in the {@link CommandTable}.
```

Comment:
I guess if we have only one command table, we don't really need to mention it here.